Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -44,6 +44,14 @@
         "Icon": "/SamplePages/HeaderedTextBlock/HeaderedTextBlock.png"
       },
       {
+        "Name": "MasterDetailsView",
```

Comment:
Damn, I can't get this through VS, hence the formatting of original file seems to be wrong, every operation you make in VS will change it. Therefore I'm trying with Notepad, now with online editor.